### PR TITLE
Revert "console autodetection"

### PR DESCRIPTION
**Pull request by spadgett**
This reverts commit 6555bedcd7d182f219cf9a605d41ce07a24a8ce3.

/assign @deads2k 

Requires https://github.com/openshift/origin-web-catalog/pull/642

Follow on openshift-ansible and origin PRs will remove the console RBAC template, which isn't needed anymore.

### CODE REVIEW

**Comment by deads2k** [comment]
I'm fine with it.  You'll want to be sure that regular users can see these resources.

**Comment by spadgett** [comment]
Yeah, we're already requesting `clusterserviceclasses` in the console. We can check if any have `spec.clusterServiceBrokerName` with `template-service-broker`.

PRs for origin and openshift-ansible:

https://github.com/openshift/origin/pull/18580
https://github.com/openshift/openshift-ansible/pull/7120

**Comment by spadgett** [comment]
Console changes have merged.

/hold cancel

File: pkg/assets/handlers.go [hunk 1]
```diff
@@ -188,16 +188,12 @@ func addExtensionStylesheets(content []byte, extensionStylesheets []string) []by
 
 var versionTemplate = template.Must(template.New("webConsoleVersion").Parse(`
 window.OPENSHIFT_VERSION = {
-  openshift: "{{ .OpenShiftVersion | js}}",
```
Comment (deads2k) [inline pkg/assets/handlers.go:191]: you no longer need to expose this?

**Comment by deads2k** [comment]
question.  lgtm otherwise

File: pkg/assets/handlers.go (hunk 1 above)
Comment (spadgett) [reply pkg/assets/handlers.go:191]: Not needed anymore. We had to switch the request to the client anyway to fix a bug: 

https://github.com/openshift/origin-web-console/pull/2734

**Comment by spadgett** [comment]
> Stage RECORD THE STARTING METADATA [00h 05m 01s] failed.

/retest

**Pull request merged**